openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/enricher/RouteURL.java

```diff
@@ -9,11 +9,7 @@
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
- * OpenShiftRouter
- * <p/>
- * Used with @ArquillianResource URL objects. Provides a URL to the OpenShift
- * router configured on the ce-cube extension, u    e.g.
- * arq.extension.openshift.routerHost, or through arquillian.xml using routerHost parameter, setting the hostname appropriately.
+ * Provides a URL of given OpenShift route.
  *
```

Comment:
I suggest you comment that, if the route is not resolvable, then the user will need to specify the routerHost containing the router ip of the Openshift cluster, otherwise addresses returned by this annotation won't be resolvable.